src/luarocks/deps.tl

```diff
@@ -740,8 +755,8 @@ local function find_lua_incdir(prefix, luaver, luajitver)
    return nil, mainerr
 end
 
-function deps.check_lua_incdir(vars)
-   if vars.LUA_INCDIR_OK == true
+function deps.check_lua_incdir(vars: {string: string}): boolean, string, string
+   if vars.LUA_INCDIR_OK == "ok" --! fix later
```

Comment:
you can search for `--! fix later` in this file and remove these; we'll keep it like this